### ISSUE TITLE
What conference is this paper submitted?

### ISSUE THREAD

**Issue by patrickphat** [tohinz/semantic-object-accuracy-for-generative-text-to-image-synthesis#9]
Hi Tohinz and your team :D 

I'm currently an undergrad and very interested in Text-to-Image synthesis. I find your paper very fascinating:
- Introduce a new metric that given calculate pair captions - generated images
- A novel model that you OP to correctly generate image objects 

The idea really makes sense to me, so I want to use this paper in my thesis, but only if this paper is published in a conference (my university's rule). Would you mind sharing me the expected date of this paper to be submitted to the conference?

Also, if I found some bugs in the code, I can join as a contributor :P Thank you!!

**Comment by tohinz** [comment]
Hi, the paper is currently under review with a journal. We're in the second round already and have addressed the reviewer's feedback from the first round. I hope to hear back from the journal soon (the reviews are done and the paper is currently with the associate editor). I will let you know once I have updates.

Sure, if you find bugs just do a pull request.

**Comment by tohinz** [comment]
Also, if you don't have time to wait until we get feedback from the journal (and I obviously don't know for sure that it will be accepted), you can also look into our previous paper published at ICLR 2019. The architecture itself is similar and the code is available here: https://github.com/tohinz/multiple-objects-gan

**Comment by patrickphat** [comment]
Hi @tohinz , I'm now looking at the published paper at ICLR 2019 and also your blog!! Also, how long does it take to train an OP-GAN?

**Comment by tohinz** [comment]
Hi, training the OP-GAN on COCO with the current parameters takes quite long (about two weeks). However, I believe you can reduce the number of parameters without much (or any) loss in the final quality. I didn't test this, but I think you could easily use half has many filters in the global and OP pathways and still get the same results. If you use fewer filters you might also be able to increase the batch size (you might have to adapt the learning rate then) to speed up the training even more.

**Comment by patrickphat** [comment]
I'll try implementing your suggestion, thank you @tohinz

**Comment by tohinz** [comment]
FYI: we just got the acceptance note from TPAMI for this paper, though it might still be a couple of weeks until it is officially available from their website.

**Comment by tohinz** [comment]
Here is the preprint + DOI on the TPAMI website: https://ieeexplore.ieee.org/document/9184960
